yarn.lock

```diff
@@ -1234,7 +1234,7 @@
     mkdirp "^1.0.4"
     rimraf "^3.0.2"
 
-"@rails/webpacker@^5.4.3":
+"@rails/webpacker@^5.4.2":
```

Comment:
why this change?

Reply:
well spotted, i shouldnt have. even tho dont think it matters, ill remove it from this pr